src/utils/nftRewards.ts

```diff
@@ -292,6 +292,31 @@ export function encodeJB721DelegatePayMetadata(
   return encoded
 }
 
+// returns list of tier Ids from a given list of indices of a given list of rewardTiers
+export function tierIdsFromTierIndices({
+  rewardTiers,
+  indices,
+}: {
+  rewardTiers: NftRewardTier[]
+  indices: number[]
+}): number[] {
+  return rewardTiers
+    .map((tier, idx) => {
+      if (indices.includes(idx)) {
+        return tier.id
+      }
+    })
+    .filter(Number) as number[]
```

Comment:
**suggestion**: does flipping the logic do the same thing?

```suggestion
  return rewardTiers
    .filter((tier, idx) => indicies.includes(idx))
    .map(tier => tier.id)
```

Reply:
Ah yes of course. Very nice